packages/cli/uni-builder/src/shared/plugins/postcss.ts

```diff
@@ -61,10 +61,19 @@ export const pluginPostcss = ({
       return mergeEnvironmentConfig(
         {
           tools: {
-            postcss: {
-              postcssOptions: {
-                plugins,
-              },
+            postcss: opts => {
+              if (typeof opts.postcssOptions === 'object') {
+                opts.postcssOptions.plugins!.push(...plugins);
+                return;
+              } else if (typeof opts.postcssOptions === 'function') {
```

Comment:
How about using `addPlugins` method? It can be simpler.

Reply:
`addPlugins` is added as extraPlugins and cannot be modified or overridden by the user